iceoryx_hoofs/test/moduletests/test_cxx_vector.cpp

```diff
@@ -144,6 +145,7 @@ TEST_F(vector_test, NewlyCreatedVectorHasSizeZero)
 TEST_F(vector_test, Capacity)
 {
     ::testing::Test::RecordProperty("TEST_ID", "e0a244d5-6e01-4cbb-9f9a-ac07cad81a5c");
+    static_assert(decltype(sut)::capacity() == 10U, "Vector capacity has to match!");
```

Comment:
Technically you are right but we require 100% MC/DC coverage in unit tests for our safety critical use case. Therefore I would just remove that line and rewrite the line below into 
```cxx
EXPECT_THAT(decltype(sut)::capacity(), Eq(10U));
```